projects/storefrontapp-e2e-cypress/cypress/integration/b2b/regression/order-history/b2b-order-history-orders-flow.e2e-spec.ts

```diff
@@ -81,4 +85,61 @@ describe('Order History with orders', () => {
     cy.get('.cx-order-history-po a').should('contain', poNumber);
     cy.get('.cx-order-history-cost-center a').should('contain', costCenter);
   });
+
+  describe('Order details - reorder', () => {
+    it('should display order details page with the reorder button', () => {
+      cy.get('.cx-order-history-value').first().click();
+      cy.get('cx-order-details-reorder div div button').should(
```

Comment:
Perhaps using this selector would make the test more resilient to UI evolution: 'cx-order-details-reorder button'